guides/v2.2/extension-dev-guide/indexing-custom.md

```diff
@@ -25,14 +25,17 @@ Your custom indexer class should implement [\Magento\Framework\Indexer\ActionInt
 
 In the `etc` directory of your module, add `indexer.xml` with the following:
 
-*  indexer ID
-*  indexer class name
-*  indexer title
-*  indexer description
-*  indexer view ID
-*  shared indexes, if any
-
-Use the optional `shared_index=` parameter to improve performance if your indexer is related to another indexer. In this [example]({{ site.mage2bloburl }}/{{ page.guide_version }}/app/code/Magento/CatalogRule/etc/indexer.xml){:target="_blank"}, if [catalog](https://glossary.magento.com/catalog) rule product needs to be reindexed, but other catalog product rule index is up-to-date, then only catalog rule product is reindexed.
+| Parameter | Required | Description |
```

Comment:
Hi @atwixfirster. Thank you for the update. That's a definitely great addition to the existing topic. Let's think together how we can separate arguments from nodes so we don't have everything mixed up in one table. 

Values for `id, class, view_id, etc` are indexer node parameters:

```xml
<indexer id="indexer_12" view_id="catalogsearch_fulltext_12" class="Magento\CatalogSearch\Model\Indexer\Fulltext">

```
At the same time, values for `title, description, structure, etc` are child nodes of the indexer node:

```xml
<indexer id="indexer_4" view_id="catalogsearch_fulltext_4" class="Magento\CatalogSearch\Model\Indexer\Fulltext">
        <title translate="true">Test Indexer Declaration</title>
        <description translate="true">Invalid field handler attribute value</description>
...
```

Both cases are still indexer parameters but they provided in a different manner. Maybe we should split your table into two separate tables in the same way? What do you think?

Reply:
@rogyar , could you please check a new version :)

Thank you so much!